src/main/java/org/fiware/mintaka/domain/query/temporal/TimeQuery.java

```diff
@@ -112,4 +112,15 @@ public String getSqlRepresentation(String dbEntityType) {
 				throw invalidTimeRelationException;
 		}
 	}
+
+	public String getDBTimeField() {
+		if(timeProperty.equals(TimeStampType.OBSERVED_AT.value())) {
```

Comment:
I think you usually add a space between if/for/while etc and the opening parenthesis.
Not for functions but for "language key-words".
That's the style guide I follow for Orion-LD (not that you need to follow the same style, just, better to be consistent)